src/cluster_info.rs

```diff
@@ -274,6 +296,24 @@ impl ClusterInfo {
             .collect()
     }
 
+    pub fn sorted_tvu_peers(&self, bank: &Arc<Bank>) -> Vec<NodeInfo> {
+        let peers = self.tvu_peers();
+        let mut peers_with_stakes: Vec<(&NodeInfo, u64)> =
```

Comment:
That can be `Vec<_>`.  The compiler is only looking for the collection type to return from `collect()`.

Reply:
True, my IDE was much better at advice when I was explicit about it. I'll fix it.